src/cmds/check_plots.py

```diff
@@ -56,12 +83,33 @@ def main():
             log.error(f"{type(e)}: {e} error in proving/verifying for plot {plot_path}")
         if total_proofs > 0:
             log.info(
-                f"{plot_path}: Proofs {total_proofs} / {args.num}, {round(total_proofs/float(args.num), 4)}"
+                f"\tProofs {total_proofs} / {args.num}, {round(total_proofs/float(args.num), 4)}"
             )
+            total_good_plots[pr.get_size()] += 1
+            total_size += plot_path.stat().st_size
         else:
+            total_bad_plots += 1
             log.error(
-                f"{plot_path}: Proofs {total_proofs} / {args.num}, {round(total_proofs/float(args.num), 4)}"
+                f"\tProofs {total_proofs} / {args.num}, {round(total_proofs/float(args.num), 4)}"
             )
+    log.info("")
+    log.info("")
+    log.info("Summary")
+    total_plots: int = sum(list(total_good_plots.values()))
+    log.info(
+        f"Found {total_plots} valid plots, total size {total_size / (1024 * 1024 * 1024 * 1024)} TB"
+    )
+    for (k, count) in sorted(dict(total_good_plots).items()):
+        log.info(f"{count} plots of size {k}")
+    grand_total_bad = total_bad_plots + len(failed_to_open_filenames)
+    if grand_total_bad > 0:
+        log.warning(f"{grand_total_bad} invalid plots")
+    if len(no_key_filenames) > 0:
+        log.warning(
+            f"There are {len(no_key_filenames)} plots with a farmer or pool public key that "
+            f"is not on this machine. The farmer private key must be in the keychain in order to "
+            f"farm them, use 'chia keys' to transfer keys. The pool public keys must be in the config.yaml"
```

Comment:
The subcommand `chia keys` doesn't follow the `chia VERB` form that all other commands follow...